src/System.IdentityModel.Tokens/Utility.cs

```diff
@@ -65,5 +65,37 @@ internal static string SerializeAsSingleCommaDelimitedString(IEnumerable<string>
 
             return sb.ToString();
         }
+
+        public static bool IsHttps(string address)
+        {
+            if (string.IsNullOrEmpty(address))
+            {
+                return false;
+            }
+
+            try
+            {
+                Uri uri = new Uri(address);
+                return IsHttps(new Uri(address));
+            }
+            catch (UriFormatException)
+            {
+                return false;
+            }
+
+        }
+
+        public static bool IsHttps(Uri uri)
+        {
+            if (uri == null)
+            {
+                return false;
+            }
+#if DNXCORE50
+            return uri.Scheme == "https"; //Uri.UriSchemeHttps is internal in dnxcore
```

Comment:
Do we know that uri.Scheme is always lower case? if not, .ToLower() is cheap and a good safety net.
Do you know why dnxcore has this internal?